manifests/prometheus-roleSpecificNamespaces.yaml

```diff
@@ -8,10 +8,12 @@ items:
   rules:
   - apiGroups:
     - ""
+    - extensions
```

Comment:
it might be cleaner to have a separate rule for ingress?

```
- apiGroups:
  - extensions
  resources:
  - ingresses
  verbs:
  - get
  - list
  - watch
- apiGroups:
  - ""
  resources:
  - services
  ...
```

Reply:
Thanks @simonpasquier I'll work on it and push updated commits with your feedback asap.